### ISSUE TITLE
New issue title tap target too small

### ISSUE THREAD

**Issue by BasThomas** [rnystrom/GitHawk#1043]
Should respond to a tap on the cell which should be 44px high.

![simulator screen shot - iphone x - 2017-11-22 at 11 06 16](https://user-images.githubusercontent.com/4190298/33121518-4cdbb6de-cf75-11e7-9545-5abbd3f2fd9f.png)
